### DISCUSSION TITLE
Alpine package

### DISCUSSION THREAD

**Discussion by Weissnix4711** [realmazharhussain/gdm-settings#154] [General]
I recently added an [alpine package](https://pkgs.alpinelinux.org/packages?name=gdm-settings&branch=edge) for gdm-settings.

Please feel free to `apk -U add gdm-settings` and let me know how it goes :smile:.

**Comment by realmazharhussain** [comment]
I myself do not use Alpine. So, I cannot test this. Sorry!

Thank you for the work though.
